Sample/SextantSample/SextantSample.csproj

```diff
@@ -21,4 +17,14 @@
     <ProjectReference Include="..\..\src\Sextant.XamForms\Sextant.XamForms.csproj" />
     <ProjectReference Include="..\..\src\Sextant\Sextant.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Update="Views\ParameterView.xaml.cs">
```

Comment:
Don't think these entries are needed, should pick it up regardless.

Reply:
Hmm.  That seems to be an issue with a files that shouldn't have been pushed.  I was moving between branches.